### PR TITLE
🐛  Replace all instances of 'context' with 'ctx'

### PR DESCRIPTION
**Pull request by Isaiah15414**
#### Summary
Context is now a keyword due to the 0.11.0 Typst update.

#### Checklist

- [x] I have fully tested all of the added features
- [x] I have fully documented all of the relevant code

### CODE REVIEW

**Comment by Isaiah15414** [comment]
I forgot to commit the documentation changes ;_;

**Comment by BattleCh1cken** [comment]
fixes #36

File: docs.typ [hunk 1]
```diff
@@ -113,29 +113,28 @@ Themes are stored as dictionaries with a set number of fields.
...
-  [ A function that returns a frontmatter entry. Must take context and body as
-    input. ],
+  [ A function that returns a frontmatter entry. Must take ctx and body as input. ],
 )
 #def-arg(
   "body-entry",
   [`<function>`],
   default: none,
-  [ A function that returns a body entry. Must take context and body as input. ],
+  [ A function that returns a body entry. Must take ctx body as input. ],
```
Comment (BattleCh1cken) [inline docs.typ]:
```suggestion
  [ A function that returns a body entry. Must take ctx and body as input. ],
```

**Review by BattleCh1cken — APPROVED** [review]
LGTM

**Pull request merged**